rust-manifest/src/config.rs

```diff
@@ -9,19 +10,7 @@ pub const DEFAULT_CONFIG_VERSION: &'static str = "1";
 #[derive(Clone, Debug)]
 pub struct Config {
     pub config_version: String,
-    pub remote: Option<ConfigRemote>,
-    pub install: ConfigInstall,
```

Comment:
I am slightly dubious about reusing `config.toml` to store the current installation state - it was intended just to be a specification of how the installation should be configured (eg. paths, remote, etc.) if that makes sense?